users/users.go

```diff
@@ -0,0 +1,94 @@
+package users
+
+import (
+	"time"
+
+	"github.com/RoadTripppin/wazzup/helpers"
+	"github.com/RoadTripppin/wazzup/interfaces"
+	"github.com/dgrijalva/jwt-go"
+	"golang.org/x/crypto/bcrypt"
+)
+
+func prepareToken(user *interfaces.User) string {
+	tokenContent := jwt.MapClaims{
+		"user_id": user.ID,
+		"expiry": time.Now().Add(time.Minute * 60).Unix(),
+	}
+	jwtToken := jwt.NewWithClaims(jwt.GetSigningMethod("HS256"), tokenContent)
+	token, err := jwtToken.SignedString([]byte("TokenPassword"))
+	helpers.HandleErr(err)
+
+	return token
+}
+
+func prepareResponse(user *interfaces.User) map[string]interface{} {
+	responseUser := &interfaces.ResponseUser{
+		ID: user.ID,
+		Username: user.Username,
+		Email: user.Email,
+	}
+
+	var token = prepareToken(user);
+	var response = map[string]interface{}{"message": "all is fine"}
+	response["jwt"] = token
+	response["data"] = responseUser
+
+	return response
+}
+
+func Login(username string, pass string) map[string]interface{} {
```

Comment:
As discussed while creating user stories, the user logs in with their email id and password. We do not have a username.